motioneye/templates/main.html

```diff
@@ -1257,7 +1257,7 @@
         <img class="background-logo" src="{{static_path}}img/motioneye-logo.svg" onmousedown="return false;">
         <div class="page-container"></div>
         <div class="footer">
-            <div class="copyright-note">copyright &copy; The motionEye Team</div>
+            <div class="copyright-note">copyright &copy; <a href="https://github.com/orgs/motioneye-project/people">The motionEye Team</a></div>
```

Comment:
```suggestion
            <div class="copyright-note">copyright &copy; <a href="https://github.com/orgs/motioneye-project/people" target="_blank" rel="noopener">The motionEye Team</a></div>
```
Making this a link in a new browser tab 🙂.